src/dashboard/src/pages/Jobs/index.tsx

```diff
@@ -82,6 +88,20 @@ const Jobs: FunctionComponent = () => {
   const history = useHistory()
   const { hash } = useLocation()
   const { clusterId } = useParams<RouteParams>()
+  const [amlUrl, setAmlUrl ] = useState('');
+  const fetchAmlUrl = '/api/clusters'
+  const requestAmlUrl = useFetch(fetchAmlUrl)
+  const fetchAml = async () => {
+    for (var i in clusters) {
+      const { amlPortal } = await requestAmlUrl.get(`/${clusters[i].id}`)
+      if (amlPortal != null && amlPortal != '') {
+        setAmlUrl(amlPortal)
```

Comment:
It seems we are fetching `amlPortal` of all clusters and overwrite them one by one